lib/model/seed_gen_model.dart

```diff
@@ -0,0 +1,18 @@
+class SeedGenModel {
+  final String seed;
+  final String deviceId;
+  final int time;
+  SeedGenModel({
+    required this.seed,
+    required this.deviceId,
+    required this.time,
+  });
+
+  factory SeedGenModel.fromJson(Map<String, dynamic> json) {
+    return SeedGenModel(
+      seed: json['body']['seed'],
+      deviceId: json['body']['id'],
+      time: json['body']['time']
+    );
+  } 
```

Comment:
**question:** are you familiar with JSON serialization tools?

Reply:
Are you referring to Automated serialization, with help of json_serializable dependencies?